src/components/ClientProjectsList.js

```diff
@@ -14,6 +14,7 @@ const ClientProjectsList = ({
 }) => {
 
     const { loading, error, data, networkStatus } = useQuery(GET_CLIENT_INFO, {
+        fetchPolicy: "cache-and-network",
```

Comment:
@Ovifer13 be sure to check your linter is properly set up I think @mecoccaro might have some recent experience with this if you have trouble

Double quote strings should be caught by the linter but always use single quotes of tildes 

```
'cache-and-network'
`cache-and-network`
```